src/styles/sidebar/components/thread.scss

```diff
@@ -0,0 +1,52 @@
+@use "../../variables" as var;
+
+.thread {
+  display: flex;
+
+  &--reply {
+    margin-top: 0.5em;
+    padding-top: 0.5em;
+  }
+
+  &__collapse {
+    margin: 0.25em;
+    margin-top: 0;
+    cursor: auto;
+    border-left: 1px dashed var.$grey-3;
+
+    &:hover {
+      border-left: 1px dashed var.$grey-4;
+    }
+
+    .is-collapsed & {
+      border-left: none;
+    }
+  }
+
+  // TODO These styles should be consolidated with other `Button` styles
```

Comment:
Please allow some loosey-gooseyness here as there is design iteration hot on the heels of this PR.

Reply:
Yes YES YES. This is not for permanent.